jq-defect-2020-1/README.md

```diff
@@ -0,0 +1,72 @@
+# jq Use-After-Free
+
+[JQ](https://github.com/stedolan/jq) is a command line utility and popular C
+library for parsing and manipulating JSON.
```

Comment:
Let's include information about the bug/possible "damage", mention responsible disclosure (and how its now fixed), and give credit to harrison with a link to his twitter or something.